src/NSubstitute/NSubstitute.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;netstandard2.0;net45;net46;net5.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net45;net46;net5.0;net6.0</TargetFrameworks>
```

Comment:
This is a breaking change and because of it we would have to release major. Can we keep it for now?

Reply:
ok, fixed